test/libsolidity/SemanticTest.cpp

```diff
@@ -164,6 +165,36 @@ map<string, Builtin> SemanticTest::makeBuiltins()
 				return toBigEndian(h256(ExecutionFramework::setAccount(accountNumber).asBytes(), h256::AlignRight));
 			}
 		},
+		{
+			"bytecode",
+			[this](FunctionCall const& _call) -> optional<bytes>
+			{
+				bool result{false};
+				optional<regex> regexPattern;
+				soltestAssert(_call.arguments.parameters.size() == 1, "Bytecode pattern string expected.");
+				string pattern{_call.arguments.parameters.at(0).rawString};
```

Comment:
The code is not wrong but I personally think that the assignment form makes it easier to see it as just initializing a variable with a copy.

```suggestion
				bool result = false;
				optional<regex> regexPattern;
				soltestAssert(_call.arguments.parameters.size() == 1, "Bytecode pattern string expected.");
				string pattern = _call.arguments.parameters.at(0).rawString;
```